### DISCUSSION TITLE
Error while building Grafana locally

### DISCUSSION THREAD

**Discussion by a-imran99** [grafana/grafana#42978] [General]
Okay, I have been trying to run grafana locally and when I try to build the backend by using the 'make run' command. The backend build fails. 
the error is as follows:
$ make run
[Bra] 12-10 19:19:20 [ INFO] App Version: 0.4.3.1020
[Bra] 12-10 19:19:20 [ INFO] Following directories are monitored:
-> C:\Users\ADMIN\Documents\Grafana\Beak_Grafana
-> $WORKDIR/pkg
-> $WORKDIR/public/views
-> $WORKDIR/conf
-> $WORKDIR/pkg/api
-> $WORKDIR/pkg/api/avatar
-> $WORKDIR/pkg/api/datasource
------------------------- [ ---SOME MORE LINES --- ]-------------------------------------
-> $WORKDIR/conf/provisioning/access-control
-> $WORKDIR/conf/provisioning/dashboards
-> $WORKDIR/conf/provisioning/datasources
-> $WORKDIR/conf/provisioning/notifiers
-> $WORKDIR/conf/provisioning/plugins
open C:\Users\ADMIN\AppData\Local\Temp\go-build1837957682\b001\exe\build.exe: Operation did not complete successfully because the file contains a virus or potentially unwanted software.
[Bra] 12-10 19:19:23 [ WARN] Fail to execute command: go [run build.go -dev build-cli] - exit status 1

This shows that the build.exe is probably an virus and my windows defender is blocking this file to be opened.

This seems to be identified as Trojan:script/Wacatac.B!ml by windows defender.

Is this a false positive??

 can anyone help this out?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
